assets/pickup-availability.js

```diff
@@ -57,6 +57,7 @@ if (!customElements.get('pickup-availability')) {
 
       document.body.appendChild(sectionInnerHTML.querySelector('pickup-availability-drawer'));
 
+      if (!this.querySelector('button')) return;
```

Comment:
We could use an existing pattern for better consistency:
```
const button = this.querySelector('button');
if (button) button.removeEventListener('click', this.onClickRefreshList);
```
Also, `this.querySelector('button')` is queried multiple times in this JS file, could we take this opportunity to add a `if (button)` condition for each instance?